MergeCalendarsTogether.gs

```diff
@@ -336,119 +339,118 @@ function ParseEvent (calendarObj, event) {
-        .some(origin => origin.address !== calendar.address && ExistsInOrigin(origin, calendar, mergedEvent));
+        .some(origin => origin.address !== calendar.address && ExistsInOrigin_(origin, calendar, mergedEvent))
       if (!primaryFound || mergedEvent.isDuplicate) {
-        let calendarRequests = payloadSets[calendar.address] || [];
+        let calendarRequests = payloadSets[calendar.address] || []
         calendarRequests.push({
           method: 'DELETE',
           endpoint: `${ENDPOINT_BASE}/${calendar.address}/events/${mergedEvent.id}`,
           summary: mergedEvent.summary, // Only used in debugging statements
-        });
-        payloadSets[calendar.address] = calendarRequests;
+        })
+        payloadSets[calendar.address] = calendarRequests
       }
-    });
-  });
+    })
+  })
 
   Object.keys(payloadSets).forEach(address => {
-    const calendarRequests = payloadSets[address];
+    const calendarRequests = payloadSets[address]
     if (!(calendarRequests || []).length) {
-      log.info(`No events to modify for ${address}.`);
+      log.info(`No events to modify for ${address}.`)
       return
     }
     if (!DEBUG_ONLY) {
       const result = new BatchRequest({
         batchPath: 'batch/calendar/v3',
         requests: calendarRequests,
-      });
+      })
       if (!result.getResponseCode || result.getResponseCode() !== 200) {
         log.info(result)
       } else {
         log.debug('RESULT: ', result.toString(), '; ', result.getResponseCode() )
-        log.info(`${calendarRequests.length} events modified for ${address}:`);
+        log.info(`${calendarRequests.length} events modified for ${address}:`)
       }
     } else {
-      log.debug(`${calendarRequests.length} events would have been modified for ${address}:`);
+      log.debug(`${calendarRequests.length} events would have been modified for ${address}:`)
     }
     const loggable = calendarRequests
       .map(({method, endpoint, summary}) => ({method, endpoint, summary}))
-    log.info(`Requests for ${address}`, JSON.stringify(loggable, null, 2));
-  });
+    log.info(`Requests for ${address}`, JSON.stringify(loggable, null, 2))
+  })
 }
 
 if (typeof module !== 'undefined') {
   module.exports = {
-    GetStartEndDates,
-    ExistsInOrigin,
-    ExistsInDestination,
+    GetStartEndDates_,
+    ExistsInOrigin_,
+    ExistsInDestination_,
     MERGE_PREFIX,
     DESC_NOT_COPIED_MSG,
-    SortEvents,
+    SortEvents_,
     IGNORE_LIST_REGEXES,
-    IsOnIgnoreList,
-    IsOnObfuscateList,
+    IsOnIgnoreList_,
+    IsOnObfuscateList_,
     OBFUSCATE_LIST_REGEXES,
     SUMMARY_NOT_COPIED_MSG,
     LOC_NOT_COPIED_MSG,
     SYNC_DAYS_IN_PAST,
     SYNC_DAYS_IN_FUTURE,
-    COPY_SELF_ATTENDANCE_STATUS,
-    AttendeeSelfStatusMatches,
-    GetAttendeeSelf,
-    ParseEvent,
-    GenerateCreatePayload,
-    ShouldObfuscate,
+    AttendeeSelfStatusMatches_,
+    GetAttendeeSelf_,
+    ParseEvent_,
+    GenerateCreatePayload_,
+    ShouldObfuscate_,
```

Comment:
is the trailing underscore pattern to indicate that it's a private function?

Reply:
It is! Functions with trailing underscore will *not* show up in the "Run" dropdown in the scripts interface, so I renamed the helper / private / internal functions!